test/Interop/Cxx/static/static-member-var-module-interface.swift

```diff
@@ -0,0 +1,7 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=StaticMemberVar -I %S/Inputs -source-filename=x -enable-cxx-interop | %FileCheck %s
+
+// CHECK: struct WithStaticAndInstanceMember {
+// CHECK:   init(myInstance: Int32)
+// CHECK-NOT:   init(myInstance: Int32, myStatic: Int32)
+// CHECK-NOT:   init(myStatic: Int32, myInstance: Int32)
+// CHECK: }
```

Comment:
Nit: another way to check this would be to say check-next.

Reply:
Fixed